<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PavilionLYX-PC" name="com.gruc.test.TestModulesLogin" tests="1" failures="1" timestamp="23 Feb 2016 05:56:37 GMT" time="44.580" errors="0">
  <testcase name="testModulesLogin" time="44.580" classname="com.gruc.test.TestModulesLogin">
    <failure type="java.lang.AssertionError" message="Current Activity is not MainActivity. expected:&lt;com.gobalroam.android.grapi.ui.MainActivity&gt; but was:&lt;com.gobalroam.android.grapi.ui.LoginActivity&gt;">
      <![CDATA[java.lang.AssertionError: Current Activity is not MainActivity. expected:<com.gobalroam.android.grapi.ui.MainActivity> but was:<com.gobalroam.android.grapi.ui.LoginActivity>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at com.gruc.modules.Login.testLogin(Login.java:65)
	at com.gruc.test.TestModulesLogin.testModulesLogin(TestModulesLogin.java:22)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- testModulesLogin -->
</testsuite> <!-- com.gruc.test.TestModulesLogin -->
